src/json.rs

```diff
@@ -27,7 +27,7 @@ struct Diagnostic {
     rendered: Option<String>,
```

Comment:
`rendered` field is unused, should I remove it?

Reply:
I never answered your question here. I'm about to release 0.3.8 and wondering if this warning should be removed?

Reply:
I guess it occurs in rustc/compiletest as well?

Reply:
https://github.com/rust-lang/rust/blob/master/src/tools/compiletest/src/json.rs#L21-L27

It's gone in rustc compiletest, should be removed.

Reply:
Hmmm, ok thanks for checking that, I'll remove it and release 0.3.9 with that if the need arises